pages/landing/patcher/index.html

```diff
@@ -42,33 +44,55 @@
 
 how_it_works:
   - heading: Install the Patcher GitHub App
-    image: /assets/img/infrastructure-cube-icons/infrastructure-design-your-stack.png
+    image: /assets/img/landing-page-patcher/github-step-1.png
     image_width: 379px
     image_side: right
     body: |
       It takes just a few clicks to add Patcher to your GitHub repos. GitLab and BitBucket support is coming soon. You can get started completely for free.
 
   - heading: Patcher discovers your dependencies automatically
-    image: /assets/img/infrastructure-cube-icons/infrastructure-design-your-stack-grunty.png
+    tab_block: site.data.patcher-landing-page-tabs-dependencies-list
     image_side: left
     body: |
       Patcher will automatically discover dependencies for popular languages and tools such as JavaScript, Ruby, Java, Python, Go, Terraform, and Docker. You can also add special comments to your code to help Patcher discover dependencies in non-standard locations (e.g., in a Bash script).
 
   - heading: Maintainers release new versions and patches
-    image: /assets/img/infrastructure-cube-icons/infrastructure-expert-support.png
-    image_width: 418px
+    precode: |
+      <figure><figcaption>terraform-to-12.patch</figcaption>
+        <pre>
+          <code class="language-yml">
+      code_transformations:
+        # Set the required_version parameter in TF { ... } blocks to 0.12
+        - update_required_version_to_tf_12
+        # Update all Gruntwork modules to versions that work with TF 0.12
+        - update_module_versions
```

Comment:
NIT: make these two comments a tiny bit shorter so no horizontal scrollbar is needed.